src/Swashbuckle.AspNetCore.Swagger/SwaggerMiddleware.cs

```diff
@@ -15,14 +17,24 @@ public class SwaggerMiddleware
         private readonly RequestDelegate _next;
         private readonly SwaggerOptions _options;
         private readonly TemplateMatcher _requestMatcher;
+        private readonly ISwaggerDocumentSerializer _swaggerDocumentSerializer;
 
         public SwaggerMiddleware(
             RequestDelegate next,
-            SwaggerOptions options)
+            SwaggerOptions options) : this(next, options, null)
+        {
+        }
+
+        public SwaggerMiddleware(
+            RequestDelegate next,
+            SwaggerOptions options,
+            IServiceProvider serviceProvider)
         {
             _next = next;
             _options = options ?? new SwaggerOptions();
             _requestMatcher = new TemplateMatcher(TemplateParser.Parse(_options.RouteTemplate), new RouteValueDictionary());
+
```

Comment:
Minor nit: I don't think there's a need for this assignment to be separated from the other three.

```suggestion
```

Reply:
Yeah, I think it is leftover from removing the comment... when the comment was there it looked better with a whitespace...